### PR TITLE
Test diagrame first scope

### PR DESCRIPTION
**Pull request by pierre-yves-monnet**
# Camunda Community Hub Pull Request Template
Test game for the migration
Different test game + automaton API to creates data in a C7 engine, ready to be migrate

## Description
Differentes test games, to cover all main use case to migrate a C7 server to a C8 server.
Each test game contains
- a description
- the BPMN process
- an Automator-process scenario, to produce process instances in the expected activity

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an existing open issue)
- [X] New feature (non-breaking change which adds functionality to an extension)
- [ ] Breaking change (fix or feature that would cause existing functionality of an extension to change)
- [ ] Documentation update (changes made to an existing piece of documentation)

## Checklist:
- [X] My code adheres to the syntax used by this extension.
- [ ] My pull request requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **Camunda Community Hub** documentation.
- [X] I have read the **Pull Request Process** documentation.
- [ ] I have added or suggested tests to cover my changes suggested in this pull request.
- [X] All new and existing CI/CD tests passed.
- [ ] I will /assign myself this issue, and add the relevant [issue labels] to it if they are not automatically generated by Probot.
- [ ] I will tag @camunda-community-hub/devrel in a new comment on this issue if 30 days have passed since my pull request was opened and I have not received a response from the extension's maintainer.

### CODE REVIEW

File: test-example-diagrams/src/main/resources/merging-inclusive/MergingInclusive.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Merging Inclusive
```
Comment (jonathanlukas) [inline test-example-diagrams/src/main/resources/merging-inclusive/MergingInclusive.md:1]: Looking at the diagram, the title should be **Merging Parallel**?

File: test-example-diagrams/src/main/resources/multiinstances-callactivity/MultiInstancesCallActivity.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Multi Instance Service Task
```
Comment (jonathanlukas) [inline test-example-diagrams/src/main/resources/multiinstances-callactivity/MultiInstancesCallActivity.md:1]: There are some loop cardinalities in the model. They do exist in Camunda 8 atm.

File: test-example-diagrams/src/main/resources/multiinstances-service-task/MultiInstancesServiceTask.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Multi Instance Service Task
```
Comment (jonathanlukas) [inline test-example-diagrams/src/main/resources/multiinstances-service-task/MultiInstancesServiceTask.md:1]: Also here, loop cardinalities are not possible at the moment in C8.

File: test-example-diagrams/src/main/resources/multinstances-usertask/MultiInstancesUserTask.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Multi instances User Task
```
Comment (jonathanlukas) [inline test-example-diagrams/src/main/resources/multinstances-usertask/MultiInstancesUserTask.md:1]: Loop Cardinality not supported in c8

File: test-example-diagrams/src/main/resources/simple-service-task/SimpleServiceTask.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Timer event
```
Comment (jonathanlukas) [inline test-example-diagrams/src/main/resources/simple-service-task/SimpleServiceTask.md:1]: Here, the header should be **Simple Service Task**

File: test-example-diagrams/src/main/resources/timer-event/TimerEvent.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Receive Event
```
Comment (jonathanlukas) [inline test-example-diagrams/src/main/resources/timer-event/TimerEvent.md:1]: Here, the header should be **Timer Event**

File: test-example-diagrams/src/main/resources/variables/Variables.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Call Activity
```
Comment (jonathanlukas) [inline test-example-diagrams/src/main/resources/variables/Variables.md:1]: Here, the header should be **Variables**

File: test-example-diagrams/src/main/resources/multiinstances-callactivity/MultiInstancesCallActivity.md (hunk 2 above)
Comment (jonathanlukas) [inline test-example-diagrams/src/main/resources/multiinstances-callactivity/MultiInstancesCallActivity.md:1]: The header should be **Multi Instance Call Activity**

**Review by jonathanlukas — CHANGES_REQUESTED** [review]
There are some things to change (mostly headers that are wrong)

File: test-example-diagrams/src/main/resources/merging-inclusive/MergingInclusive.md (hunk 1 above)
Comment (pierre-yves-monnet) [reply test-example-diagrams/src/main/resources/merging-inclusive/MergingInclusive.md:1]: No, this is an Inclusive gateway here in the diagram ( ? )
image

File: test-example-diagrams/src/main/resources/multiinstances-service-task/MultiInstancesServiceTask.md (hunk 3 above)
Comment (pierre-yves-monnet) [reply test-example-diagrams/src/main/resources/multiinstances-service-task/MultiInstancesServiceTask.md:1]: I split in 2 test games

File: test-example-diagrams/src/main/resources/simple-service-task/SimpleServiceTask.md (hunk 5 above)
Comment (pierre-yves-monnet) [reply test-example-diagrams/src/main/resources/simple-service-task/SimpleServiceTask.md:1]: Done

File: test-example-diagrams/src/main/resources/timer-event/TimerEvent.md (hunk 6 above)
Comment (pierre-yves-monnet) [reply test-example-diagrams/src/main/resources/timer-event/TimerEvent.md:1]: Done

File: test-example-diagrams/src/main/resources/variables/Variables.md (hunk 7 above)
Comment (pierre-yves-monnet) [reply test-example-diagrams/src/main/resources/variables/Variables.md]: Done

File: test-example-diagrams/src/main/resources/multiinstances-callactivity/MultiInstancesCallActivity.md (hunk 2 above)
Comment (pierre-yves-monnet) [reply test-example-diagrams/src/main/resources/multiinstances-callactivity/MultiInstancesCallActivity.md:1]: Done

File: test-example-diagrams/src/main/resources/merging-inclusive/MergingInclusive.md (hunk 1 above)
Comment (jonathanlukas) [reply test-example-diagrams/src/main/resources/merging-inclusive/MergingInclusive.md:1]: Sorry, I just looked at the png, and there is a parallel gateway on merging:

image

**Review by jonathanlukas — CHANGES_REQUESTED** [review]
The png in merging inclusive needs to be renewed

**Review by jonathanlukas — APPROVED** [review]

**Pull request merged**